weejson/xml/test/src/com/rallyhealth/weejson/v1/xml/XmlTests.scala

```diff
@@ -101,21 +101,25 @@ class XmlTests extends FreeSpec with Matchers with Inside with TypeCheckedTriple
       assert(fromBytes == fromInputStream)
     }
     "shortcuts" - {
-      "positive" - {
+      "positive" in {
         assertResult(Map("a" -> Str("1")))(readXml("<a>1</a>").obj) // everything's a string
         assertResult(1)(readXml("<a>1</a>")("a").str.toInt) // everything's a string
       }
-      "negative" - {
-        intercept[Exception] {
-          val x = readXml("1") // parses as object with nameless attribute - {"":"1"}
-          val y = x.transform(ToXml.string) // puts data in an empty tag - <root><>1</></root>
-          FromXml(y).transform(Value) // cant parse empty tag - // WstxUnexpectedCharException: Unexpected character '>' (code 62) in content after '<' (malformed start element?).
-        }
-        intercept[Exception] {
```

Comment:
@russellremple, these don't throw anymore. Is that okay?

Reply:
seems fine